javascriptlib/src/mill/javascriptlib/RsWithExpressModule.scala

```diff
@@ -0,0 +1,31 @@
+package mill.javascriptlib
+
+import mill.*
+import os.*
+
+trait RsWithExpressModule extends TypeScriptModule {
```

Comment:
Why do we need this? Why can't we just use the normal `TypeScriptModule#run` that the `server` module already comes with?

Reply:
- we need to bundle our React application and supply derived bundled path that would be served up by the defined server
- we export to this trait since we intend to have this pattern re-used

Reply:
moved to resources